README.md

```diff
@@ -52,8 +52,16 @@ Make use of the many generators for code, try `ember help generate` for more det
 
 ### Running Tests
 
-* `ember test`
-* `ember test --server`
+We use [ember-exam](https://github.com/trentmwillis/ember-exam) for running tests since it allows for parallel testing, randomized orders, and other neat configurations. You can check out the repository readme there to see the available options. A few handy ones to use when running tests locally are outlined below.
+
+* `ember exam` will run the tests in order
+* `ember exam --split=3 --weighted --parallel` will run tests in 3 PhantomJS instances in parallel with an equal split. This will be a little faster but unstable on systems with low RAM.
+* `ember exam --random` will run the tests in a random order
+* `ember exam --filter='acceptance' will only run acceptance tests. Other options here include Unit and Integration. You can also exclude
```

Comment:
Looks like you're missing a trailing ` here and that the sentence just sort of trails off. I would also include the actual usage for `='unit'` and `='integration'`.